package.json

```diff
@@ -10,19 +10,21 @@
     "build": "tsc -b tsconfig.build.json",
     "watch": "tsc -b tsconfig.build.json -w",
     "lint": "npm run lint --workspaces",
-    "test": "npm run test --workspace=langium  --workspace=domainmodel",
+    "test": "jest --coverage",
```

Comment:
Coverage measurement seems not to work:
```
----------|---------|----------|---------|---------|-------------------
File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
----------|---------|----------|---------|---------|-------------------
All files |       0 |        0 |       0 |       0 |                   
----------|---------|----------|---------|---------|-------------------
```

Reply:
I don't have a clue what changes this. Tried different things like adding --no-cache or --watchAll. Found those solution on [stackoverflow](https://stackoverflow.com/questions/57451028/jest-finds-tests-but-doesnt-collect-coverage).
Created a new issue for that: https://github.com/langium/langium/issues/252